src/StoreKeeper/WooCommerce/B2C/Tools/Attributes.php

```diff
@@ -486,6 +480,10 @@ public static function importAttribute(
             $attribute_id = WordpressExceptionThrower::throwExceptionOnWpError(
                 wc_update_attribute($existingAttribute->id, $update_arguments)
             );
+
+            // Manually update attribute as wc_update_attribute reverts
+            // attribute type to select when it does not exist
+            self::updateAttributeType($existingAttribute);
```

Comment:
can you add the comment that we do it because some plugins require admin level, and than it breaks?

Reply:
Added comment regarding woocommerce swatch.